dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.mpHealth.4.0.ee.10.0.mp-6.0.feature

```diff
@@ -0,0 +1,8 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
```

Comment:
Required because in `dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.mpHealth.4.0.ee-10.0.feature` if the feature list is set to start `mpConfig-3.0` and tolerates `3.1` issues arise when starting `microProfile-6.1`.The server will start to start `mpConfig-3.0` when `3.1` is required (even though 3.1 is tolerated).

If you explicitly set configure `mpHealth-4.0` and `mpConfig-3.1` it will work fine, but using `microProfile-6.1` will fail.

The opposite effect happens if you swap it to start `mpConfig-3.1` and tolerates `3.0`. It will run fine if `mpConfig-3.0` is set, but will fail if `microProfile-6.0` is used. It will cause it to load mpConfig-3.1 (and not accept that 3.0 is tolerated).

Reply:
Yeah this is expected because if you have public tolerates you need to include it in the public feature coming from the private features which would be a problem because you cannot tolerate 3.0 and tolerate 3.1 universally in mpHealth 4.0.  You need to handle older versions of config as well and those do not start with the same feature name due to the switch in feature prefix.